src/app/app.component.ts

```diff
@@ -1,10 +1,81 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit  } from '@angular/core';
+import { discardPeriodicTasks } from '@angular/core/testing';
+import { Injectable, Pipe, PipeTransform } from '@angular/core';
+import{SearchFilterPipe} from './filter.pipe'
 @Component({
   selector: 'app-root',
+
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
+
 })
-export class AppComponent {
-  title = 'app';
+export class AppComponent implements OnInit{
+  tasks: Task[] = [];
+  newTask = "  ";
+  id = 1;
+  itemFilter: any = { text: '' };
+  tasksDone = [];
+   title:String;
+constructor() { 
+  this.title = " ";
+  this.tasks = []
+}
+
+ngOnInit() {
+}
+
+
+
+  pushTask = function () {
+    // jesli ten element jest pusty dodaje nowy task
+    if (this.newTask) {
+      const newTaskObject: Task = {
+        title: this.newTask,
+        date: new Date(),
+        done: false
+      };
+      this.tasks.push(newTaskObject);
+      this.newTask = " ";
+    }
+  }
+
+  // usuwanie
+  removeTask = function (index) {
+    this.tasks.splice(index, 1);
+  }
+  removeTask1 = function (index) {
```

Comment:
Nie nazywaj w ten sposób metod. Nazwy powinny coś mówić a konkretnie powinny dokładnie opisywać to co to coś robi.

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/2d8f41b40c7bf09298a62f2a286fa0e41d65a614